### PR TITLE
Feature / HOLO 1466 Add hToken Support

### PR DESCRIPTION
**Pull request by BravoNatalie**
## Describe Changes

I made this better by doing ...

Add support to ERC20 CrossChainMessageType:
- Add HLG hardcoded addresses
- Change  `protocol-events-map` file

## Checklist before requesting a review

- [x] I have performed a self-review of my code
- [x] Code styles have been enforced
- [x] I have checked eslint

### CODE REVIEW

File: src/utils/contracts.ts [hunk 1]
```diff
@@ -99,6 +99,19 @@ export const METADATA_RENDERER_ADDRESS: {[key in Environment]: string} = {
   [Environment.mainnet]: '0x60B839C2f7dBa29eB93b094067E6C87067d1B3df'.toLowerCase(),
 }
 
+export const ERC20_HLG_ADDRESSES: {[key in Environment]: string} = {
```
Comment (alexanderattar) [inline src/utils/contracts.ts:102]: Where did you grab these addresses from? They're not matching what I have in the protocol repo

**Review by alexanderattar — CHANGES_REQUESTED** [review]
I think the HLG addresses are not right. Let's double check. The address you have for mainnet isn't deployed so I believe these were earlier deployments

File: src/utils/contracts.ts (hunk 1 above)
Comment (BravoNatalie) [reply src/utils/contracts.ts:102]: I'm not entirely sure, but I believe Angello provided me with those addresses a few months back.

**Comment by alexanderattar** [comment]
Mainnet HLG - `0x740df024CE73f589ACD5E8756b377ef8C6558BaB`
Testnet HLG - `0x09506B435782714EABC66F34256B28661990900d`
Develop HLG - `0x700bFAE1E03Af52CD1EeBD42C763CEA125e6CA1F`

**Review by alexanderattar — APPROVED** [review]

**Review by feliperodriguess — APPROVED** [review]

**Pull request merged**